charts/operator/templates/clusterrole.yaml

```diff
@@ -17,250 +17,257 @@ metadata:
-      - get
-      - list
-      - watch
-  - apiGroups:
-      - ""
-    resources:
-      - pods
-    verbs:
-      - delete
-      - get
-      - list
-      - update
-      - watch
-  - apiGroups:
-      - ""
-    resources:
-      - pods/finalizers
-    verbs:
-      - update
-  - apiGroups:
-      - ""
-    resources:
-      - pods/status
-    verbs:
-      - patch
-      - update
-  - apiGroups:
-      - ""
-    resources:
-      - secrets
-    verbs:
-      - create
-      - get
-      - list
-      - patch
-      - update
-      - watch
-  - apiGroups:
-      - ""
-    resources:
-      - serviceaccounts
-    verbs:
-      - create
-      - get
-      - list
-      - patch
-      - update
-      - watch
-  - apiGroups:
-      - ""
-    resources:
-      - services
-    verbs:
-      - create
-      - get
-      - list
-      - patch
-      - update
-      - watch
-  - apiGroups:
-      - networking.k8s.io
-    resources:
-      - ingresses
-    verbs:
-      - create
-      - delete
-      - get
-      - list
-      - patch
-      - update
-      - watch
-  - apiGroups:
-      - policy
-    resources:
-      - poddisruptionbudgets
-    verbs:
-      - create
-      - delete
-      - get
-      - list
-      - patch
-      - update
-      - watch
-  - apiGroups:
-      - rbac.authorization.k8s.io
-    resources:
-      - clusterrolebindings
-      - clusterroles
-    verbs:
-      - create
-      - get
-      - list
-      - patch
-      - update
-      - watch
-  - apiGroups:
-      - redpanda.vectorized.io
-    resources:
-      - clusters
-    verbs:
-      - create
-      - delete
-      - get
-      - list
-      - patch
-      - update
-      - watch
-  - apiGroups:
-      - redpanda.vectorized.io
-    resources:
-      - clusters/finalizers
-    verbs:
-      - update
-  - apiGroups:
-      - redpanda.vectorized.io
-    resources:
-      - clusters/status
-    verbs:
-      - get
-      - patch
-      - update
-  - apiGroups:
-      - redpanda.vectorized.io
-    resources:
-      - consoles
-    verbs:
-      - create
-      - delete
-      - get
-      - list
-      - patch
-      - update
-      - watch
-  - apiGroups:
-      - redpanda.vectorized.io
-    resources:
-      - consoles/finalizers
-    verbs:
-      - update
-  - apiGroups:
-      - redpanda.vectorized.io
-    resources:
-      - consoles/status
-    verbs:
-      - get
-      - patch
-      - update
+- apiGroups:
+    - ""
+  resources:
+    - persistentvolumes
+  verbs:
+    - delete
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - apps
+  resources:
+    - deployments
+  verbs:
+    - create
+    - delete
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - apps
+  resources:
+    - statefulsets
+  verbs:
+    - create
+    - delete
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - apps
+  resources:
+    - statefulsets/status
+  verbs:
+    - patch
+    - update
```

Comment:
The `statefulsets/status` role rules are defined both in ClusterRole and Role. My assumption was that we stick to namespaced operator in v2. With introduction of decommission in helm chart I don't understand why we would need cluster scope permission for `statefulse/status`. Could you help me understand that?